src/MLJDecisionTreeInterface.jl

```diff
@@ -107,6 +117,27 @@ function MMI.predict(m::DecisionTreeClassifier, fitresult, Xnew)
     return MMI.UnivariateFinite(classes_seen, scores)
 end
 
+MMI.reports_feature_importances(::Type{<:DecisionTreeClassifier}) = true
+
+function MMI.feature_importances(m::DecisionTreeClassifier, fitresult, report)
+    # generate feature importances for report
+    if m.feature_importance == :impurity
+        feature_importance_func = DT.impurity_importance
+    elseif m.feature_importance == :split
+        feature_importance_func = DT.split_importance
+    end
+
+    tree = fitresult[1]
+    features = fitresult[end]
+    fi = feature_importance_func(tree)
```

Comment:
Let's make use of normalized feature importances similar to sklearn.
```suggestion
    fi = feature_importance_func(tree, normalize=true)
```